developer_manual/contributing-guide.rst

```diff
@@ -0,0 +1,80 @@
+Contributing to LibreSign
+==================
+
+Overview
+--------
+
+Welcome
+^^^^^^^
+
+
+We really appreciate everyone who would like to contribute to the LibreSign project! There are many ways to contribute, including writing code, filing issues on GitHub, helping people, assisting in triaging, reproducing, or fixing bugs that people have filed, and enhancing our documentation. Also giving a star to the project is a really good way to help and donate.
+
+Before you get started, we encourage you to read our code of conduct, which describes our community norms:
+
+    * 1. `Our code of conduct < />`__, which explicitly stipulates that everyone must be gracious, respectful, and professional. This also documents our conflict resolution policy and encourages people to ask questions.
```

Comment:
The link here is to https://github.com/LibreSign/libresign/blob/main/CODE_OF_CONDUCT.md